config/owasp/suppressions.xml

```diff
@@ -29,13 +29,6 @@
     <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
     <cve>CVE-2020-13943</cve>
   </suppress>
-  <suppress>
-    <notes><![CDATA[
-   file name: launchdarkly-java-server-sdk-5.1.1.jar (shaded: com.google.guava:guava:28.2-jre)
-   ]]></notes>
-    <packageUrl regex="true">^pkg:maven/com\.google\.guava/guava@.*$</packageUrl>
-    <cve>CVE-2020-8908</cve>
-  </suppress>
```

Comment:
CVE removed, hoping also removed from confluence page list

Reply:
Removed entry for CRD in Confluence page.However there are other repos too so still you can find this CVE for others

Reply:
ok please merge then